### ISSUE TITLE
Vysor is not installing on Android N preview

### ISSUE THREAD

**Issue by bnacey** [koush/vysor.io#128]
I know this is still brand new, but just to raise awareness I'm submitting this issue.

I've installed Android N preview on a Nexus 9. 
Vysor can find the device. After launching the viewer window, it never gets past "Installing Vysor APK..."

**Comment by nealsanche** [comment]
I also just ran into this. Not sure where to look to get more information for you.

**Comment by cd1** [comment]
I'm also having the same problem on my Nexus 6. Here's some log from logcat which might be related to the issue:

```
03-15 16:44:25.263 17928 17928 D AndroidRuntime: >>>>>> START com.android.internal.os.RuntimeInit uid 2000 <<<<<<
03-15 16:44:25.268 17928 17928 D AndroidRuntime: CheckJNI is OFF
03-15 16:44:25.330 17928 17928 I art     : JIT created with initial_capacity=64KB, max_capacity=64MB, compile_threshold=500, save_profiling_info=1
03-15 16:44:25.338 17928 17928 D ICU     : No timezone override file found: /data/misc/zoneinfo/current/icu/icu_tzdata.dat
03-15 16:44:25.378 17928 17928 I Radio-JNI: register_android_hardware_Radio DONE
03-15 16:44:25.406 17928 17928 D AndroidRuntime: Calling main entry com.android.commands.pm.Pm
03-15 16:44:25.419 17928 17928 I art     : System.exit called, status: 1
03-15 16:44:25.419 17928 17928 I AndroidRuntime: VM exiting with result code 1.
```

I wanted to show off some new features of Android N using Vysor but I can't :disappointed:

**Comment by koush** [comment]
should be fixed.

**Comment by nealsanche** [comment]
Yep. Definitely.

On Sun, May 8, 2016, 3:50 PM Koushik Dutta <notifications@github.com> wrote:

> Closed #128 <https://github.com/koush/vysor.io/issues/128>.
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
> <https://github.com/koush/vysor.io/issues/128#event-653816999>
>

-- 
Robots and Pencils Inc | 1507 14th Ave SW Calgary AB T3C 0W4
You can unsubscribe from the Robots and Pencils mailing lists by clicking 
here. <https://robotsandpencils.com/contact/unsubscribe.html>
